terminal_velocity/urwid_ui.py

```diff
@@ -306,6 +308,15 @@ def quit(self):
 
         raise urwid.ExitMainLoop()
 
+    def execute_hook(self, hook, param):
+        """Execute the defined hook
+
+        For the time being only the post hook has been tested
+
+        """
+        if hook in self.hooks != "" and os.path.isfile(self.hooks[hook]):
```

Comment:
This is a bit odd. `hook in self.hooks` is a boolean expression, it will always equal ` True` or `False`, never `""`, so ` if hook in self.hooks != ""` will always be `True`.

I think this might do what you wanted: `if self.hooks.get(hook) and ...`.